spec/BowlingSpec.js

```diff
@@ -0,0 +1,64 @@
+describe('Bowling Scorecard', function() {
+  var scorecard, frame;
+
+  beforeEach(function() {
+    frame = new Frame();
+  });
+
+  describe('bowls a single ball and –', function() {
+    describe('keeps track of the frame’s score, which:', function() {
+      it('is initally 0', function() {
+        expect(frame.frameScore).toEqual(0);
+      });
+
+      it('(potentially) increases as balls are bowled', function() {
+        frame.bowl();
```

Comment:
So frame is both bowling *and* keeping track of its `frameScore` - this looks like something that might be better for two separate prototypes